x-pack/plugins/audit_trail/server/client/audit_trail_client.ts

```diff
@@ -4,8 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
```

Comment:
@thomheymann were you able to explore this approach? We'll need to solve for backwards compatability before we can merge

Reply:
No longer an issue since audit trail service has now been moved into security plugin and can be configured / enabled the same way as the legacy plugin